src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,42 @@
+package calculator;
+
+import java.util.List;
+
+public class Calculator {
+
+    public double calculate(List<Integer> operands, List<Character> operators) {
+        double result = operands.get(0);
+        int numOfOperations = operators.size();
+        int times = 0;
+
+        while (times < numOfOperations) {
+            result = fourFundamentalArithmeticOperations(result, operators.get(times), operands.get(++times));
+        }
+
+        return result;
+    }
+
+    public double fourFundamentalArithmeticOperations(double tempResult, char operator, double operand) {
+        double result;
+
+        switch (operator) {
+            case '+':
+                result = tempResult + operand;
+                break;
+            case '-':
+                result = tempResult - operand;
+                break;
+            case '*':
+                result = tempResult * operand;
+                break;
+            case '/':
+                result = tempResult / operand;
+                break;
+            default:
+                result = 0;
+                break;
+        }
+
+        return result;
+    }
```

Comment:
switch 문을 활용해 결과를 깔끔하게 처리하셨군요! 👍 
하지만 return 문을 활용하면 break 삭제할 수 있어서 코드 길이를 줄일 수 있을 것 같아요.
```suggestion
    public calculateExpression(double tempResult, char operator, double operand) {
        
        switch (operator) {
            case '+':
                return tempResult + operand;
            case '-':
                return tempResult - operand;
            case '*':
                return tempResult * operand;
            case '/':
                return tempResult / operand;
        }

        return 0;
    }
```
한 가지 덧붙이자면 보통 switch 문은 함수의 길이를 지나치게 길게 만들기 때문에 기피하고는 한답니다. (클린코드, 47p)
책의 내용을 참고해서 해석하자면 함수의 길이가 길다는 의미는
 '더하기', '빼기', '곱하기', '나누기' 등의 여러 작업을 동일 함수 내에서 진행하고 있고 이는 곧 코드를 변경할 이유가 여러 가지라고 합니다.

하지만 switch 문 자체를 완전히 배제할 수는 없기 때문에 책에서 제안하는 방법은 다형성을 활용해서 switch 문을 풀어내는 것입니다.  자세한 내용은 책을 참조하시면 좋을 것 같습니다!